src/api/consumer/lib/add.js

```diff
@@ -21,6 +23,7 @@ export async function addMany(
   components: AddProps[],
   consumerPath: string = process.cwd()
 ): Promise<AddActionResults[]> {
+  const configureConsumer = consumerPath !== process.cwd();
```

Comment:
Please add somewhere a comment about why you need this.. it won't be obvious for someone who read this code.